lib/graphql/schema.ts

```diff
@@ -169,18 +231,50 @@ export type TransactionsConnection = {
 
 export type TransactionsConnectionEdge = {
    __typename?: 'TransactionsConnectionEdge',
-  node: TransactionListItem,
+  node: Transaction,
   cursor: Scalars['String'],
 };
 
 export type Transfer = {
    __typename?: 'Transfer',
-  status: Scalars['String'],
-  purpose: Scalars['String'],
+  status: TransferStatus,
+  amount: Scalars['Int'],
+  note?: Maybe<Scalars['String']>,
+  id: Scalars['String'],
+  recipient: Scalars['String'],
+  iban: Scalars['String'],
+  e2eId?: Maybe<Scalars['String']>,
+};
+
+export enum TransferStatus {
```

Comment:
🤔 

build is failing because of these duplicated entries: 
```
##[error]lib/graphql/schema.ts(250,3): error TS2300: Duplicate identifier 'Created'.
##[error]lib/graphql/schema.ts(251,3): error TS2300: Duplicate identifier 'Created'.
##[error]lib/graphql/schema.ts(252,3): error TS2300: Duplicate identifier 'Authorized'.
##[error]lib/graphql/schema.ts(253,3): error TS2300: Duplicate identifier 'Authorized'.
##[error]lib/graphql/schema.ts(254,3): error TS2300: Duplicate identifier 'Confirmed'.
##[error]lib/graphql/schema.ts(255,3): error TS2300: Duplicate identifier 'Confirmed'.
##[error]lib/graphql/schema.ts(256,3): error TS2300: Duplicate identifier 'Booked'.
##[error]lib/graphql/schema.ts(257,3): error TS2300: Duplicate identifier 'Booked'.
##[error]lib/graphql/transaction.ts(4,3): error TS2305: Module '"./schema"' has no exported member 'TransactionListItem'.
```